modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -18,6 +18,19 @@ To start using PTP fast event notifications for a network interface in your clus
 
 . Modify the default PTP Operator config to enable PTP fast events.
 
+.. Get the name of the `transportHost` in the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get ptpoperatorconfig -n openshift-ptp -o jsonpath='{ .items[*].spec.ptpEventConfig.transportHost }'
+----
++
+.Example output
+[source,terminal]
+----
+http://ptp-event-publisher-service-compute-1-example.openshift-ptp.svc.cluster.local:9043
```

Comment:
Use the following http transport example, so that it is clear the node name is used.
http://ptp-event-publisher-service-node1.openshift-ptp.svc.cluster.local:9043